src/components/ContactDetails.vue

```diff
@@ -570,7 +623,31 @@ export default {
 
 			return ''
 		},
-
+		profilePageLink() {
+			// just for testing this is not fully correct
+			return ''
+		},
+		emailAddressProperties() {
+			return this.localContact.properties.find(property => property.name === 'email')
+		},
+		emailAddress() {
+			return this.emailAddressProperties.getFirstValue()
+		},
+		phoneNumber() {
+			return this.phoneNumberProperties.getFirstValue()
```

Comment:
this is where https://github.com/nextcloud/contacts/pull/3401#issuecomment-1612876117 happens. `this.phoneNumberProperties` returns `undefined` if there is no property in `this.localContact.properties` with name===tel